src/include/homestore/btree/btree.hpp

```diff
@@ -52,7 +77,9 @@ class Btree {
 #ifndef NDEBUG
     std::atomic< uint64_t > m_req_id{0};
 #endif
-
+#ifdef _PRERELEASE
+    BTREE_FLIPS m_flips;
+#endif
```

Comment:
Why should btree maintain a separate vector of flips? Why can't the caller directly set the flip?